app/ts/library/flashbots.ts

```diff
@@ -105,7 +106,12 @@ export async function sendBundle(bundle: Bundle, targetBlock: bigint, fundingAmo
 		maxBaseFee,
 	)
 
-	const payload = JSON.stringify({ jsonrpc: '2.0', method: 'eth_sendBundle', params: [{ txs, blockNumber: `0x${targetBlock.toString(16)}`, revertingTxHashes: [] }] })
+	const payload = JSON.stringify({
+		jsonrpc: '2.0',
+		method: 'eth_sendBundle',
+		id: bundleId++,
```

Comment:
This could just be hard coded to `1` if we wanted.  What you have is also fine though.